Cargo.toml

```diff
@@ -38,3 +38,5 @@ default = []
 # Only used for running the integration tests
 integration-test = ['dep:flate2','dep:tar','dep:minreq']
 
+[lints.clippy]
+uninlined_format_args = "allow"
```

Comment:
Instead of putting this in cargo toml please add it in the lib.rs `#![allow(clippy::uninlined_format_args)]`. 

We are already putting all allows in the lib, so lets not break that pattern.